tools/who_what_benchmark/whowhatbench/utils.py

```diff
@@ -22,3 +24,12 @@ def new_randn_tensor(
 
 def patch_diffusers():
     torch_utils.randn_tensor = new_randn_tensor
+
+
+def add_device_specific_properties(device, ov_config):
+    properties = ov.Core().get_property(device, ov.properties.supported_properties)
+
+    if "ACTIVATIONS_SCALE_FACTOR" in properties:
+        ov_config["ACTIVATIONS_SCALE_FACTOR"] = 8.0  # FP16 overflow issue
```

Comment:
should we add this option to pipelines itself? E.g. to VAE autoencoder (as it's done in optimum-intel)

Reply:
> should we add this option to pipelines itself? E.g. to VAE autoencoder (as it's done in optimum-intel)

I didn't get your point. This property is bypassed in every pipeline if GPU is specified as the inference device. We also agreed that GPU will pull this option from rt_info so that this option can be added to the IR during the model export from Optimum.

Reply:
if `ov_config` already contains `ACTIVATIONS_SCALE_FACTOR`, I think we need to preserve that value